File: cpp/31d_1360A_MinimalSquare.cpp
#include<bits/stdc++.h>

int main(){
    int t, a, b, side;
    std::vector<int> ans;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>a>>b;
        side = std::max((2 * (std::min(a,b))), (std::max(a,b)));
        ans.push_back((side * side));
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/31d_1360A_MinimalSquare.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        a, b = input().split()
        a, b = int(a), int(b)
        side = max((2 * min(a,b)), max(a,b))
        ans.append((side * side))
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()